src/App.tsx

```diff
@@ -1,18 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const items = getNumbers(1, 42).map((n) => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [page, setPage] = useState(1);
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+
+  const pagination = () => {
+    const pagin = [];
+
+    for (let i = 1; i <= Math.ceil(items.length / itemsPerPage); i += 1) {
+      pagin.push(i);
+    }
+
+    return pagin;
+  };
```

Comment:
please rename this function smth like `getPaginationList` and move it outside of the component because every re-render it will be a new function